fastlane/Fastfile

```diff
@@ -408,6 +409,7 @@ platform :ios do
       include_bitcode: false,
       include_symbols: true,
       clean: true,
+      xcargs: "-skipPackagePluginValidation",
```

Comment:
<table data-meta="generated_by_pr-check">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Automattic/pocket-casts-ios/blob/2daecc5ee5662b7c036d698dfd74684833a0d748/fastlane/Fastfile#L412"></span>Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.</td>
    </tr>
  </tbody>
</table>